packages/manager/apps/hub/src/dashboard/dashboard.controller.js

```diff
@@ -1,31 +1,50 @@
-import { TRACKING_PREFIX, HIT_PREFIX } from './dashboard.constant';
+import {
+  TRACKING_PREFIX,
+  HIT_PREFIX,
+  TRACKING_PREFIX_POPUP,
+} from './dashboard.constant';
 
 export default class DashboardController {
   /* @ngInject */
-  constructor(coreURLBuilder, coreConfig, atInternet) {
+  constructor(coreURLBuilder, coreConfig, atInternet, $window) {
     this.coreURLBuilder = coreURLBuilder;
     this.user = coreConfig.getUser();
     this.atInternet = atInternet;
+    this.$window = $window;
     this.TRACKING_PREFIX = TRACKING_PREFIX;
     this.HIT_PREFIX = HIT_PREFIX;
+    this.TRACKING_PREFIX_POPUP = TRACKING_PREFIX_POPUP;
   }
 
   $onInit() {
     this.availableSiretBanner = false;
+    this.availableSiretPopup = false;
+
+    const userSiretFR =
+      !this.user.companyNationalIdentificationNumber &&
+      this.user.legalform === 'corporation' &&
+      this.user.country === 'FR';
 
     this.isDisplayableHubBanner.then((data) => {
-      this.availableSiretBanner =
-        data &&
-        !this.user.companyNationalIdentificationNumber &&
-        this.user.legalform === 'corporation' &&
-        this.user.country === 'FR';
+      this.availableSiretBanner = data && userSiretFR;
+
       if (this.availableSiretBanner) {
         this.atInternet.trackPage({
           name: TRACKING_PREFIX,
           type: 'navigation',
         });
       }
     });
+    this.isDisplayableHubPopup.then((data) => {
+      this.availableSiretPopup = data && userSiretFR;
+      if (this.availableSiretPopup) {
+        this.atInternet.trackPage({
+          name: TRACKING_PREFIX_POPUP,
+          type: 'navigation',
+        });
+      }
+    });
```

Comment:
Maybe you have possibility to merge this call with your first call to avoid 2 calls to the feature-flipping function